quizzes/08_spread.js

```diff
@@ -0,0 +1,83 @@
+console.log(
+  // getMax([7, 8, 5, 9, 20, 7])
```

Comment:
I'd prefer to not pass arguments to the quiz functions and just have them declare the variables inside the function itself.